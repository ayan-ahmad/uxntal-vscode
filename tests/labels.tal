@label
&sublabel

@label-label
@label-la/bel ( `/` highlighted )
@la/bel-la/bel ( Second `/` not highlighted )
@
@/
@//

&sublabel-sublabel
&sublabel-subla/bel ( `/` not highlighted )
&
&/
