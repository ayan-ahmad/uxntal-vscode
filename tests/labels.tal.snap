>@label
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.tag.uxntal
>&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^ source.tal entity.name.tag.uxntal
>
>@label-label
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^^^^ source.tal entity.name.tag.uxntal
>@label-la/bel ( `/` highlighted )
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^ source.tal entity.name.tag.uxntal
#         ^ source.tal punctuation.rune.uxntal
#          ^^^ source.tal entity.name.tag.uxntal
#             ^ source.tal
#              ^ source.tal comment.uxntal
#               ^ source.tal
#                ^^^ source.tal comment.uxntal
#                   ^ source.tal
#                    ^^^^^^^^^^^ source.tal comment.uxntal
#                               ^ source.tal
#                                ^ source.tal comment.uxntal
>@la/bel-la/bel ( Second `/` not highlighted )
#^ source.tal punctuation.rune.uxntal
# ^^ source.tal entity.name.tag.uxntal
#   ^ source.tal punctuation.rune.uxntal
#    ^^^^^^^^^^ source.tal entity.name.tag.uxntal
#              ^ source.tal
#               ^ source.tal comment.uxntal
#                ^ source.tal
#                 ^^^^^^ source.tal comment.uxntal
#                       ^ source.tal
#                        ^^^ source.tal comment.uxntal
#                           ^ source.tal
#                            ^^^ source.tal comment.uxntal
#                               ^ source.tal
#                                ^^^^^^^^^^^ source.tal comment.uxntal
#                                           ^ source.tal
#                                            ^ source.tal comment.uxntal
>@
#^ source.tal punctuation.rune.uxntal
>@/
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
>@//
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
#  ^ source.tal entity.name.tag.uxntal
>
>&sublabel-sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^^^^^^^^^^ source.tal entity.name.tag.uxntal
>&sublabel-subla/bel ( `/` not highlighted )
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^^^^^^^^^^^ source.tal entity.name.tag.uxntal
#                   ^ source.tal
#                    ^ source.tal comment.uxntal
#                     ^ source.tal
#                      ^^^ source.tal comment.uxntal
#                         ^ source.tal
#                          ^^^ source.tal comment.uxntal
#                             ^ source.tal
#                              ^^^^^^^^^^^ source.tal comment.uxntal
#                                         ^ source.tal
#                                          ^ source.tal comment.uxntal
>&
#^ source.tal punctuation.rune.uxntal
>&/
#^ source.tal punctuation.rune.uxntal
# ^ source.tal entity.name.tag.uxntal
>